examples/cpp_execution/text_inference_using_cpp.py

```diff
@@ -0,0 +1,236 @@
+    2. Check if exported ONNX file already exists, if true, jump to compilation -> execution, else
+    3. Check if HF model exists in cache, if true, start transform -> export -> compilation -> execution, else,
+    4. Download HF model -> transform -> export -> compile -> execute
+    ``Mandatory`` Args:
+        :model_name (str): Hugging Face Model Card name, Example: ``gpt2``
+        :num_cores (int): Number of cores to compile model on.
+    ``Optional`` Args:
+        :device_group (List[int]): Device Ids to be used for compilation. If ``len(device_group) > 1``, multiple Card setup is enabled. ``Defaults to None.``
+        :prompt (str): Sample prompt for the model text generation. ``Defaults to None.``
+        :prompts_txt_file_path (str): Path to txt file for multiple input prompts. ``Defaults to None.``
+        :aic_enable_depth_first (bool): Enables ``DFS`` with default memory size. ``Defaults to False.``
+        :mos (int): Effort level to reduce the on-chip memory. ``Defaults to -1.``
+        :batch_size (int): Batch size to compile the model for. ``Defaults to 1.``
+        :full_batch_size (int): Set full batch size to enable continuous batching mode. ``Default to None``
+        :prompt_len (int): Prompt length for the model to compile. ``Defaults to 32.``
+        :ctx_len (int): Maximum context length to compile the model. ``Defaults to 128.``
+        :generation_len (int): Number of tokens to be generated. ``Defaults to False.``
+        :mxfp6 (bool): Enable compilation for MXFP6 precision. ``Defaults to False.``
+        :mxint8 (bool): Compress Present/Past KV to ``MXINT8`` using ``CustomIO`` config. ``Defaults to False.``
+        :local_model_dir (str): Path to custom model weights and config files. ``Defaults to None.``
+        :cache_dir (str): Cache dir where downloaded HuggingFace files are stored. ``Defaults to None.``
+        :hf_token (str): HuggingFace login token to access private repos. ``Defaults to None.``
+
+    .. code-block:: bash
+
+        python -m examples.text_inference_from_cpp OPTIONS
+
+    """
+    cache_dir = check_and_assign_cache_dir(local_model_dir, cache_dir)
+    tokenizer = load_hf_tokenizer(
+        pretrained_model_name_or_path=(local_model_dir if local_model_dir else model_name),
+        cache_dir=cache_dir,
+        hf_token=hf_token,
+    )
+
+    qeff_model = AutoModelForCausalLM.from_pretrained(model_name)
+    generated_qpc_path = qeff_model.compile(
+        num_cores=14,
+        mxfp6=True,
+        device_group=[0],
+    )
+
+    #########
+    # Execute
+    #########
+    cloud_ai_100_exec_kv(
+        tokenizer=tokenizer,
+        qpc_path=generated_qpc_path,
+        prompt_len=prompt_len,
+        prompt=prompt,
+        device_id=device_group,
+        prompts_txt_file_path=prompts_txt_file_path,
+        generation_len=generation_len,
+        full_batch_size=full_batch_size,
+    )
+
+
+def read_prompts_txt_file(prompts_txt_file_path: str):
```

Comment:
Please add try except block to this function in original text_generation_interface and import from there.